frontend/src/components/common/Filter/components/FilterContent/components/FilterMenu/index.tsx

```diff
@@ -66,46 +58,45 @@ export default function FilterMenu({
   }, []);
 
   return (
-    <MenuWrapper menuWidth={menuWidth} ref={menuRef}>
-      <Menu>
-        {/* Optional search bar */}
-        {isSearchable && (
-          <FilterSearch
-            searchValue={searchValue}
-            setSearchValue={setSearchValue}
-          />
-        )}
+    <Menu
```

Comment:
Core SDS or Mui components replacing Blueprint here and elsewhere. Props such as `isMenuDivided` and `scrollable` are now passed into the parent component; the parent components target the styles of child components in an effort to improve rendering performance.